Gemfile

```diff
@@ -15,6 +15,7 @@ gem "omniauth", '= 1.8.1'                                          # omniauth is
 gem "omniauth-slack", '= 2.3.0'                                    # the slack strategy for omniauth and supports the sign in with slack approval flow
 gem 'configatron'                                                  # add multi-environment yaml settings
 gem 'countries', '~> 2.1.4'                                        # all sorts of useful information about every country
+gem 'httparty', '~> 0.17.0'                                        # http requests for callbacks to Slack slash command API
```

Comment:
Does the API request we need exist as a subset of `gem 'slack-ruby-client', '~> 0.11'` instead of adding httparty?

Reply:
From what I can tell, no.  Despite the fact that it indicates you can respond to slash commands in the README, the documentation is non-existent on how to do that.  There are no in-code references to response type, response url, "in_channel" (which is a response type needed to reply to a slash command) or even any comment/documentation on Slash commands being used in this way.

The RealTime Client seems to only respond to Slack API events (and sending a slash command is not one of those).  The Web Client does not handle sending responses back to direct messages or sending a message back via any segment of the response URL.  I've tested multiple implementations of this to no avail.  My guess is despite the documentation, it doesn't have robust slash command handling, at least from what I can find.